new.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
it's a verb to preceed a noun (the type of thing you're creating). it reads quite nicely - ./new.sh plugin alethzero MyPlugin